### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by gkkulik** [OleVik/grav-skeleton-timeline#2]
Dude, you really need to work on your documentation here. You start from the middle, not telling potential users where to begin. How do I even tell Grav which page should work as a timeline? By a code? If yes, then what code is it? Why is there no working example's configuration as a point of reference? You show some screenshots but how do I even get there? If I want create a new page in the admin area, I don't see the "Default ordering" or "Default order direction" fields. Does it mean my plugin does not work? This is horrible.

**Comment by OleVik** [comment]
From [Settings and Usage](https://github.com/OleVik/grav-skeleton-timeline#settings-and-usage), which I admit should come earlier in the README:

> Each timeline is structured with a Header (timeline.md, Timeline-template) and Events (timeline_event.md, Timeline Event-template).

Any Page that you create in Grav with the filename `timeline.md` is the wrapper for the Timeline, and will take any Page created with the filename `timeline_event.md` as its Events. Consider the Constantinian Dynasty, in [this folder](https://github.com/OleVik/grav-skeleton-timeline/tree/master/pages/timeline/Dominate/Constantinian). The initial [timeline.md](https://github.com/OleVik/grav-skeleton-timeline/blob/master/pages/timeline/Dominate/Constantinian/timeline.md) creates the Page at https://olevik.me/staging/grav-skeleton-timeline/timeline/dominate, and all the Pages below it are Events - each with their own [timeline_event.md](https://github.com/OleVik/grav-skeleton-timeline/blob/master/pages/timeline/Dominate/Constantinian/julian/timeline_event.md), like Julian at https://olevik.me/staging/grav-skeleton-timeline/timeline/dominate/constantinian/julian.

From the [Description](https://github.com/OleVik/grav-skeleton-timeline#description) at the top:

> A [demo is available](https://olevik.me/staging/grav-skeleton-timeline), and demo content can be found in [/pages](https://github.com/OleVik/grav-skeleton-timeline/tree/master/pages).

The plugin assumes some knowledge of basics with Grav, largely [Pages](https://learn.getgrav.org/16/content/content-pages) and [FrontMatter](https://learn.getgrav.org/16/content/headers). If you are using the Admin-plugin, you will need to create a Page using the Timeline or Timeline Event template, that is what is demonstrated in the README.

Be aware of the [Requirements](https://github.com/OleVik/grav-skeleton-timeline#requirements) to use the plugin: PHP's Intl-extension must be installed. Beyond that, I just tested with the latest Grav v1.7.0-rc.20 + Admin v1.10.0-rc.20 , and navigated to the Pages-administration:

![image](https://user-images.githubusercontent.com/974717/102470264-0c234280-4054-11eb-9c09-ca0256d9e39d.png)

Clicking the Add Page button yields:

![image](https://user-images.githubusercontent.com/974717/102470471-4ee51a80-4054-11eb-9675-cf36618db3f7.png)

I'll accept that I can make the steps even clearer in the initial README. I will not accept that there is a lack of configuration of point of reference, as that is this entire repository. If you find it horribly difficult to use the plugin, please give me an outline of what you would expect to be explained up front, and I shall endeavour to improve the documentation.

**Comment by gkkulik** [comment]
Thank you. Sorry if I sounded too harsh. My frustration got the better of me. Found the issue: my PHP installation lacked the Intl extenstion and after adding it, Timeline and Timeline event showed up in the Page template dropdown menu.

What I missed last night is the Intl requirement sentence that is put below a bolded statement and above a header. I might have missed it because it's kind of invisible. Obviously it might have been also my fault but it wouldn't do any harm if it stood out more.

Then what helped me here was that you showed me that after a correct installation Timeline and Timeline event have the status of a page template. I could not understand if they were to be embedded into a normal page or something else entirely. After clearing that up everything is pretty straightforward. Thank you again for your help.
